platform/platform-filter-delegate/src/main/java/org/codice/ddf/platform/filter/delegate/FilterInjector.java

```diff
@@ -78,13 +94,60 @@ public void event(ServiceEvent event, Map<BundleContext, Collection<ListenerInfo
     }
   }
 
+  public void init() {
+    executorService.schedule(this::checkForMissedServletContexts, 1, TimeUnit.SECONDS);
+  }
+
+  public void destroy() {
+    executorService.shutdownNow();
+  }
+
+  private void checkForMissedServletContexts() {
+    try {
+      BundleContext context = getContext();
+      if (context == null) {
+        return; // bundle is probably refreshing
+      }
+      Collection<ServiceReference<ServletContext>> references =
+          getContext().getServiceReferences(ServletContext.class, null);
```

Comment:
❓Shouldn't we use `context` here instead of calling `getContext()` again and risking a `null` back?